elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -566,9 +571,19 @@ export class PeerPool extends EventEmitter {
 		})();
 	}
 
-	// TODO: Protect peers by netgroup
 	private _selectPeersForEviction(): Peer[] {
 		const peers = [...this.getPeers(InboundPeer)];
+		// Cannot predict which netgroups will be protected
+		const filteredPeersByNetgroup = this._peerPoolConfig.netgroupProtectionRatio
```

Comment:
shouldn't `netgroupProtectionRatio` always exist?

Reply:
User has the option to set it to `0` which means they want to disable it.